compute_endpoint/globus_compute_endpoint/endpoint/config/model.py

```diff
@@ -98,6 +98,27 @@ class EngineModel(BaseConfigModel):
     class Config:
         validate_all = True
 
+    @root_validator(pre=True)
+    @classmethod
+    def _validate_engine_strategy(cls, values: dict):
+        engine_type = values.get("type")
+        strategy = values.get("strategy")
+        if engine_type == "GlobusComputeEngine" and isinstance(strategy, dict):
+            raise ValueError(
+                "strategy as an object is incompatible with the GlobusComputeEngine."
+                " Please update to the string value 'simple' or null."
```

Comment:
I know this is what's in the story verbatim, but I like your more explicit example below, why not write that here as well?

ie.  add "\n" "  E.g.,\n" '     strategy: "simple"'

Reply:
Works for me.